source/assets/javascripts/_gradient-background.js

```diff
@@ -1,7 +1,33 @@
-$(document).mousemove(function(e){
-  var $width = ($(document).width())/255;
-  var $height = ($(document).height())/255;
-  var $pageX = parseInt(e.pageX / $width,10);
-  var $pageY = parseInt(e.pageY / $height,10);
-  $("body").css("background", "linear-gradient(135deg, rgb(112, "+$pageX+", "+$pageY+"), rgb(255, "+$pageY+", "+$pageX+")" );
+$(function() {
+  var lastXPosition, lastYPosition;
+
+  $(document).mousemove(function(e){
+    if(lastXPosition == e.clientX && lastYPosition == e.clientY) {
+      return
+    }
+
+    lastXPosition = e.clientX
+    lastYPosition = e.clientY
+
+    console.log(e)
+    var $width = ($(document).width())/255;
+    var $height = ($(document).height())/255;
+    var $pageX = parseInt(e.clientX / $width,10);
+    var $pageY = parseInt(e.clientY / $height,10);
```

Comment:
Oh you don't know about `radix`? Neither do I really, so good question.

via: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
```
radix
An integer between 2 and 36 that represents the radix (the base in mathematical 
numeral systems) of the above mentioned string. Specify 10 for the decimal numeral 
system commonly used by humans. Always specify this parameter to eliminate 
reader confusion and to guarantee predictable behavior. Different implementations 
produce different results when a radix is not specified, usually defaulting the value to 10.
```